.setup/install_system.sh

```diff
@@ -135,6 +139,8 @@ export DAEMONCGI_GROUP=${DAEMONCGI_GROUP}
 alias submitty_help=\"echo -e '${INSTALL_HELP}'\"
 alias install_submitty='/usr/local/submitty/.setup/INSTALL_SUBMITTY.sh'
 alias submitty_install='/usr/local/submitty/.setup/INSTALL_SUBMITTY.sh'
+alias install_lichen='sudo bash /usr/local/submitty/GIT_CHECKOUT/Lichen/install_lichen.sh'
+alias lichen_install='sudo bash /usr/local/submitty/GIT_CHECKOUT/Lichen/install_lichen.sh'
```

Comment:
You don't need `sudo` here. These aliases are only available to the root user.

Reply:
I tried it without the `sudo` first and it spit out an error about missing permissions, so I added it...

Reply:
What was the error message?

Reply:
Hm, I don't remember but I can try testing it again. Btw, is the only way to test this script to do `destroy` and `up` in vagrant, or is it possible to somehow rerun it?

Reply:
You can just directly edit `/root/.bashrc` and `source /root/.bashrc` to test the aliases.